lib/location.js

```diff
@@ -97,6 +97,7 @@ var LocationState = State.extend(function (self, name, opts) {
         characters_per_page:160,
         next_text:"Next",
         previous_text:"Previous",
+        skip_text: "Skip",
```

Comment:
We need to maintain compatibility for existing applications so `skip_text` should probably default to `null` and then we should only add the skip handler and display `s. Skip` if `skip_text` is not null.